src/InvalidArgumentException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Fig\Cache;
```

Comment:
The preference on the mailing list seemed to favor reserving the Psr\ namespace for the PSRs themselves, not for any auxiliary code.  I'm inclined to agree.